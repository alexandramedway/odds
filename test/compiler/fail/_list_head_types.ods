do head([1,2,3]) || true
