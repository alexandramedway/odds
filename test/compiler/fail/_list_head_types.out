Analyzer Exception: Invalid use of binary operator '||' with types Num and Bool
